Content.Shared/Clothing/Components/NonStickSurfaceComponent.cs

```diff
@@ -0,0 +1,10 @@
+namespace Content.Shared.Clothing.Components;
+
+/// <summary>
+///     If an item has this on it reagents will not stick
+///     to it and fall to the ground (E.g lube or glue).
+/// </summary>
```

Comment:
This can stay. Though maybe be more specific like `Prevents sticky and lube from working on this entity.`